### DISCUSSION TITLE
Jekyll fora do GitHub, em servidores comuns

### DISCUSSION THREAD

**Discussion by kvnol** [frontendbr/forum#626] [Discussão]
Rép, galero.

Então, eu trampo com Jekyll em alguns mini projetinhos fora do GitHub e queria saber se vocês também trabalham dessa forma e como fazem o deploy, porque FTP não dá mais e se tem alguma forma de fazer o build direto no servidor. Alguma dica? Largar o Jekyll? rs.

**Comment by nandomoreirame** [comment]
Vc pode usar ferramentas que fazer integração continua, como TravisCI e outros... aqui na empresa usamos o Gitlab para versionar os code, então estamos usando o Gitlab CI é bem interessante... 

Da pra usar automatizar o build e o deploy com Docker tbm ^^

**Comment by kvnol** [comment]
@nandomoreirame Massa, Nando. Também uso o GitLab aqui no trampo, mas não com o TravisCI. Então o Travis conversa tanto com o GitLab quanto com o servidor "comum" (ex. Locaweb, Hostgator...).

Tem algum artigo pra indicar? :heart:

**Comment by nandomoreirame** [comment]
Então, a gente não usa o Travis aqui na empresa, eu uso por fora mesmo... usamos o GitLab CI mesmo, é dentro do próprio Gitlab... ele é bem parecido com o Travis (só que de grátis)

**Comment by nandomoreirame** [comment]
@kvnol [GitLab Continuous Integration & Deployment Pipelines](https://about.gitlab.com/features/gitlab-ci-cd/)

**Comment by filipelinhares** [comment]
Eu uso o [surge.sh](http://surge.sh/). Muito bom 😄

**Comment by barraponto** [comment]
eu faço o build na minha máquina e publico usando o netlify.
mas dá pra fazer via CI também, é coisa linda.
obs: dá pra subir o compilado via FTP sim. via linha de comando, inclusive.

**Comment by rubenmarcus** [comment]
Opa cai nesse artigo esses dias, pode ser que ajude também:
https://css-tricks.com/deployment/

**Comment by caiogaspar** [comment]
Pra uns projetos menores eu uso o https://deploybot.com/, sempre funcionou muito bem pra mim!
